src/api/spec/models/notification_rss_feed_item.rb

```diff
@@ -0,0 +1,83 @@
+require "rails_helper"
+
+RSpec.describe Notification::RssFeedItem do
+  describe '.cleanup' do
+    let(:user) { create(:confirmed_user, login: 'manolo') }
+    let(:user2) { create(:confirmed_user, login: 'manolo2') }
+    let(:group) { create(:group, users: [user, user2]) }
+    let(:unconfirmed_user) { create(:user, login: 'manolo_unconfirmed') }
+    let(:not_enough_items) { Notification::RssFeedItem::MAX_ITEMS_PER_USER }
+    let(:enough_items) { not_enough_items + 3 }
```

Comment:
Maybe call this something like `let(:greater_than_max_items_per_user)` or ``let(:greater_than_max_items)`